tests/deploy/ci-deploy.sh

```diff
@@ -39,7 +39,7 @@ github_curl() {
 
 get_latest_release_tag() {
   if [ -z "${LATEST_REL_TAG:-}" ]; then
-    LATEST_REL_TAG="$(github_curl "https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/releases/latest" | jq -r '.tag_name')"
+    LATEST_REL_TAG="v3.40.4"
```

Comment:
Assuming this is temp, to test the upgrade path

Reply:
Not temp, but I forgot to make the release branch. It's staying, but will only be in a `release-6.2.0` branch. Need to key on the right thing to upgrade from, apparently v3.40.4 is too old since it was still set to 1.30. Setting it to v3.40.8. I guess platform-apps's `release-v71` had a really stale eks reference for some reason.